addons/website_slides/static/src/scss/website_slides.scss

```diff
@@ -514,11 +514,9 @@ $line-height-truncate: 1.25em;
 // **************************************************
 .o_wslides_lesson_main {
     .o_wslides_lesson_aside {
-        .o_wslides_background_image_aside_card {
```

Comment:
Is it not a "background" image? I wouldn't rename it unless it's confusing at some level.